common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -148,6 +150,26 @@ public BrokerRequest brokerRequestFromSilentOperationParameters(@NonNull final S
         return brokerRequest;
     }
 
+    public @NonNull Bundle getRequestBundleForSsoToken(final @NonNull AcquirePrtSsoTokenCommandParameters parameters,
+                                                       final @NonNull String negotiatedBrokerProtocolVersion) {
+        Bundle requestBundle = new Bundle();
+        requestBundle.putString(AuthenticationConstants.Broker.ACCOUNT_NAME, parameters.getAccountName());
+        requestBundle.putString(AuthenticationConstants.Broker.ACCOUNT_HOME_ACCOUNT_ID, parameters.getHomeAccountId());
+        requestBundle.putString(AuthenticationConstants.Broker.ACCOUNT_LOCAL_ACCOUNT_ID, parameters.getLocalAccountId());
+        if (parameters.getAccountAuthority() != null) {
+            requestBundle.putString(ACCOUNT_AUTHORITY, parameters.getAccountAuthority());
```

Comment:
as commented in the broker pr, this should probably be a "request authority" as opposed to "account authority".

Reply:
Think I've renamed all of this.